src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcTimeline.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.timeline.Timeline
+
+@this()
+
+@(params: Timeline)(implicit messages: Messages)
+@import params._
```

Comment:
Probably worth enforcing mandatory parameters like here: https://github.com/hmrc/play-frontend-hmrc/blob/7bc0366ad080c1c24be2d6aa328721e186e8f61d/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcCharacterCount.scala.html#L29

Reply:
I'm not sure what we'd test for there, `headingLevel` has a default, `events` defaults to an empty sequence.  The `Event` requires a `title`, `time` and `content` with an optional `datetime`.